src/lib/parse/CompoundValueNode.h

```diff
@@ -127,6 +126,7 @@ class CompoundValueNode
   void                check(void);
   std::string         finish(void);
   std::string         render(Format format, const std::string& indent);
+  std::string         renderV2(void);
```

Comment:
I don't if we could call this renderSimple instead of renderV2 ... don't know if it is a good idea to create methods that refer to an specific version rather than to a functionality